AUTHORS

```diff
@@ -976,14 +982,107 @@ Igor Kalashnikov <igor.kalashnikov@me.com>
 Anna Henningsen <anna@addaleax.net>
 Amery <amery@xiangfa.org>
 James Reggio <james.reggio@gmail.com>
+Imran Iqbal <imran@imraniqbal.org>
```

Comment:
Make sure to remove the other entry with me name @ line 800, I am no longer with IBM